README.md

```diff
@@ -1 +1,265 @@
-# Final_Capstone_Front_End
\ No newline at end of file
+<a name="readme-top"></a>
```

Comment:
- [ ] This readme file is full of instructions about the backend project.

Kindly, craft a readme file suitable for the frontend project, including instructions about how to properly setup and run it. 👍